Building DAG of jobs...
Using shell: /usr/bin/bash
Provided cores: 1 (use --cores to define parallelism)
Rules claiming more threads will be scaled down.
Job stats:
job                         count
------------------------  -------
scanpy_doublet_detection        1
total                           1

Select jobs to execute...

[Thu Jul 25 03:27:10 2024]
rule scanpy_doublet_detection:
    input: data/processed/scanpy/03_qc_filtered_data.h5ad
    output: data/processed/scanpy/04_doublet_detected_data.h5ad
    log: logs/scanpy/04_doublet_detection.log
    jobid: 0
    reason: Missing output files: data/processed/scanpy/04_doublet_detected_data.h5ad
    resources: tmpdir=/tmp

/opt/miniforge3/envs/scanpy/bin/python3.9 /mnt/exfat01/projects/scRNA-0722/.snakemake/scripts/tmptfcnqzpa.04_doublet_detection.py
[Thu Jul 25 03:28:15 2024]
Error in rule scanpy_doublet_detection:
    jobid: 0
    input: data/processed/scanpy/03_qc_filtered_data.h5ad
    output: data/processed/scanpy/04_doublet_detected_data.h5ad
    log: logs/scanpy/04_doublet_detection.log (check log file(s) for error details)

RuleException:
CalledProcessError in file /mnt/exfat01/projects/scRNA-0722/scanpy.smk, line 49:
Command 'set -euo pipefail;  /opt/miniforge3/envs/scanpy/bin/python3.9 /mnt/exfat01/projects/scRNA-0722/.snakemake/scripts/tmptfcnqzpa.04_doublet_detection.py' returned non-zero exit status 1.
  File "/mnt/exfat01/projects/scRNA-0722/scanpy.smk", line 49, in __rule_scanpy_doublet_detection
  File "/opt/miniforge3/envs/scanpy/lib/python3.9/concurrent/futures/thread.py", line 58, in run
Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
Complete log: .snakemake/log/2024-07-25T032709.953964.snakemake.log
